packages/react/src/components/checkbox/checkbox.react.tsx

```diff
@@ -0,0 +1,36 @@
+import { c, CheckboxProps as BaseProps, classy, m } from '@onfido/castor';
+import { InputContainer } from '@onfido/castor-react';
+import React from 'react';
+import { withRef } from '../../utils';
+
+export const Checkbox = withRef(
+  (
+    {
+      bordered,
+      invalid,
+      children,
+      className,
+      style,
+      ...restProps
+    }: CheckboxProps,
+    ref: CheckboxProps['ref']
+  ): JSX.Element => (
+    <InputContainer {...{ bordered, className, style }}>
```

Comment:
💅🏻  [nitpick]

Spreading props `{...{ bordered, className, style }}`, although terser, gives worse errors (TypeScript, ESLint).
For instance, if `bordered` had the wrong type, the component would have a red squiggly line instead of the offending property.

![image](https://user-images.githubusercontent.com/18623773/105346168-20230c80-5bdd-11eb-9bdd-0855df76cacf.png)

This is much clearer in my opinion:

![image](https://user-images.githubusercontent.com/18623773/105346254-40eb6200-5bdd-11eb-9b8e-91ea53586dce.png)